### ISSUE TITLE
It is compatible with 3 phase?

### ISSUE THREAD

**Issue by platini76** [persuader72/silla-prism-integration#6]
hello it is possible to have compatibility with 3phase?

**Comment by persuader72** [comment]
Yes, In theory, on the official Prism manual there are no differences in the mqtt topics between 1 and 3 phases models.

If you try it let me know, if there are problems.

Regards
Stefano

**Comment by platini76** [comment]
hello Stefano, maybe italian? as me.
maybe mqtt 3 phase has info on 3 phase?

**Comment by persuader72** [comment]
Can you send me the topics'output  related to the Prism from your mqtt borker? In this way I can look for disfferences respect the single phase model.
For instance in my RaspberryPI i can obtain it with this command:
mosquitto_sub -v -h 127.0.0.1 -t "prism/#" -u ***** -P *****
I let it run at least the time need to acquire a full round of topics. (1 minute is enough)


Puoi inviarmi l'output dei topic del tuo MQTT brocker relativi alla Prism? Cosi posso vedere le differenze con il modello a singola fase.
Ad esempio nel mio RaspberryPI utilizzo questo comando:
mosquitto_sub -v -h 127.0.0.1 -t "prisma/#" -u ***** -P *****
In genere Lo lascio gireare almeno il tempo necessario per acquisire un giro completo di topic. (2 minuti dovrebbero essere sufficienti)

Regards

**Comment by platini76** [comment]
Our command line:
$ ha help
➜  / mosquitto_sub -v -h 127.0.0.1 -t "prisma/#" -u ***** -P *****
Error: Unknown option 'addon_configs'.

Use 'mosquitto_sub --help' to see usage.
➜  / 

forse c'e' qualcosa che non va
